bin/propolis-server/src/lib/serial/mod.rs

```diff
@@ -14,10 +14,10 @@ use std::sync::Arc;
 use std::time::Duration;
 
 use crate::serial::history_buffer::{HistoryBuffer, SerialHistoryOffset};
+use dropshot::WebsocketConnectionRaw;
```

Comment:
Why the `use` here, when the later references are made to `dropshot::WebsocketConnectionRaw`, rather than the bare name?

Reply:
um... bad programming? I'll fix it